### PR TITLE
zh-translation: some announcing news

### PR DESCRIPTION
**Pull request by ilylia**
ref: https://github.com/servicemesher/istio-official-translation/issues/1708
ref: https://github.com/servicemesher/istio-official-translation/issues/1555
ref: https://github.com/servicemesher/istio-official-translation/issues/1553
ref: https://github.com/servicemesher/istio-official-translation/issues/1552
ref: https://github.com/servicemesher/istio-official-translation/issues/1551
ref: https://github.com/servicemesher/istio-official-translation/issues/1547
ref: https://github.com/servicemesher/istio-official-translation/issues/1542

### CODE REVIEW

File: content/zh/news/releases/1.1.x/announcing-1.1.6/index.md [hunk 1]
```diff
@@ -12,18 +12,18 @@ aliases:
...
 {{< relnote >}}
 
-## Bug fixes
+## Bug 修复{#bug-fixes}
 
-- Fix Galley Helm charts so that the `validatingwebhookconfiguration` object can now deployed to a namespace other than `istio-system` ([Issue 13625](https://github.com/istio/istio/issues/13625)).
-- Additional Helm chart fixes for anti-affinity support: fix `gatewaypodAntiAffinityRequiredDuringScheduling` and `podAntiAffinityLabelSelector` match expressions and fix the default value for `podAntiAffinityLabelSelector` ([Issue 13892](https://github.com/istio/istio/issues/13892)).
-- Make Pilot handle a condition where Envoy continues to request routes for a deleted gateway while listeners are still draining ([Issue 13739](https://github.com/istio/istio/issues/13739)).
+- 修复 Galley Helm charts 使得 `validatingwebhookconfiguration` 可以部署到其它命名空间（非 `istio-system`）中（[Issue 13625](https://github.com/istio/istio/issues/13625)）。
+- 另外的支持 anti-affinity 的 Helm chart 修复：修复 `gatewaypodAntiAffinityRequiredDuringScheduling` 和 `podAntiAffinityLabelSelector` 匹配表达式以及修复 `podAntiAffinityLabelSelector` 的默认值（[Issue 13892](https://github.com/istio/istio/issues/13892)）。
```
Comment (gorda) [inline content/zh/news/releases/1.1.x/announcing-1.1.6/index.md]: > 另外的支持 anti-affinity 的 Helm chart 修复：

建议：为反亲和性支持提供额外的 Helm chart  修复：

File: content/zh/news/releases/1.1.x/announcing-1.1.6/index.md [hunk 2]
```diff
@@ -12,18 +12,18 @@ aliases:
...
 
-## Small enhancements
+## 小改进{#small-enhancements}
 
-- If access logs are enabled, `passthrough` listener requests will be logged.
-- Make Pilot tolerate unknown JSON fields to make it easier to rollback to older versions during upgrade.
-- Add support for fallback secrets to `SDS` which Envoy can use instead of waiting indefinitely for late or non-existent secrets during startup ([Issue 13853](https://github.com/istio/istio/issues/13853)).
+- 如果启用了访问日志，`passthrough` 侦听器的请求将被记录。
+- 使 Pilot 容忍未知的 JSON 字段，以便在升级过程中更轻松地回滚到旧版本。
+- `SDS` 增加对后备 secrets 的支持，使得 Envoy 可以使用它而不是在启动过程中无限期地等待最新或不存在的 secrets（[Issue 13853](https://github.com/istio/istio/issues/13853)）。
```
Comment (gorda) [inline content/zh/news/releases/1.1.x/announcing-1.1.6/index.md]: > secrets 

建议：secret

File: content/zh/news/releases/1.1.x/announcing-1.1.8/index.md [hunk 3]
```diff
@@ -12,26 +12,26 @@ aliases:
...
-- Fix locality load balancing issue where only one replica in a locality would receive traffic ([13994](https://github.com/istio/istio/issues/13994)).
-- Fix issue where Pilot Agent might not notice a TLS certificate rotation ([Issue 14539](https://github.com/istio/istio/issues/14539)).
-- Fix a `LuaJIT` panic in Envoy ([Envoy Issue 6994](https://github.com/envoyproxy/envoy/pull/6994)).
-- Fix a race condition where Envoy might reuse a HTTP/1.1 connection after the downstream peer had already closed the TCP connection, causing 503 errors and retries ([Issue 14037](https://github.com/istio/istio/issues/14037)).
-- Fix a tracing issue in Mixer's Zipkin adapter causing missing spans ([Issue 13391](https://github.com/istio/istio/issues/13391)).
+- 修复 CDS 集群的 `PASSTHROUGH` `DestinationRules`（[Issue 13744](https://github.com/istio/istio/issues/13744)）。
+- 使 Helm charts 中的 `appVersion` 和 `version` 字段显示正确的 Istio 版本（[Issue 14290](https://github.com/istio/istio/issues/14290)）。
+- 修复 Mixer 崩溃同时影响策略和遥测服务（[Issue 14235](https://github.com/istio/istio/issues/14235)）。
+- 修复多集群时不同集群中的两个 pods 无法共享同一个 IP 地址的问题（[Issue 14066](https://github.com/istio/istio/issues/14066)）。
```
Comment (gorda) [inline content/zh/news/releases/1.1.x/announcing-1.1.8/index.md]: > pods 

建议：pod

File: content/zh/news/releases/1.1.x/announcing-1.1.8/index.md [hunk 4]
```diff
@@ -12,26 +12,26 @@ aliases:
...
+- 修复当 Citadel 无法连接 Kubernetes API 服务时重新生成新的根 CA 导致双向 TLS 验证失败的问题（[Issue 14512](https://github.com/istio/istio/issues/14512)）。
+- 改进 Pilot 验证以拒绝相同域名的不同 `VirtualServices`，因为 Envoy 不会接受（[Issue 13267](https://github.com/istio/istio/issues/13267)）。
+- 修复了本地负载平衡问题，即本地中只有一个副本会接收流量（[13994](https://github.com/istio/istio/issues/13994)）。
```
Comment (gorda) [inline content/zh/news/releases/1.1.x/announcing-1.1.8/index.md]: > 负载平衡

建议：负载均衡

File: content/zh/news/releases/1.1.x/announcing-1.1.6/index.md (hunk 1 above)
Comment (ilylia) [reply content/zh/news/releases/1.1.x/announcing-1.1.6/index.md]: all fixed

**Pull request merged**